src/forms/PayoutDetailsForm/PayoutDetailsForm.js

```diff
@@ -114,43 +110,33 @@ const PayoutDetailsFormComponent = props => (
         })
       );
 
-      const streetAddressLabel = intl.formatMessage({
-        id: 'PayoutDetailsForm.streetAddressLabel',
-      });
-      const streetAddressPlaceholder = intl.formatMessage({
-        id: 'PayoutDetailsForm.streetAddressPlaceholder',
-      });
-      const streetAddressRequired = validators.required(
-        intl.formatMessage({
-          id: 'PayoutDetailsForm.streetAddressRequired',
-        })
-      );
-
-      const postalCodeLabel = intl.formatMessage({ id: 'PayoutDetailsForm.postalCodeLabel' });
-      const postalCodePlaceholder = intl.formatMessage({
-        id: 'PayoutDetailsForm.postalCodePlaceholder',
-      });
-      const postalCodeRequired = validators.required(
-        intl.formatMessage({
-          id: 'PayoutDetailsForm.postalCodeRequired',
-        })
-      );
-
-      const cityLabel = intl.formatMessage({ id: 'PayoutDetailsForm.cityLabel' });
-      const cityPlaceholder = intl.formatMessage({ id: 'PayoutDetailsForm.cityPlaceholder' });
-      const cityRequired = validators.required(
-        intl.formatMessage({
-          id: 'PayoutDetailsForm.cityRequired',
-        })
-      );
-
-      const showAddressFields = country && requiresAddress(country);
-
       // StripeBankAccountTokenInputField handles the error messages
       // internally, we just have to make sure we require a valid token
       // out of the field. Therefore the empty validation message.
       const bankAccountRequired = validators.required(' ');
 
+      const showPersonalIdNumber =
+        (country && stripeCountryConfigs(country).personalIdNumberRequired) ||
+        (country && stripeCountryConfigs(country).ssnLast4Required);
+
+      const personalIdNumberLabel = showPersonalIdNumber
+        ? intl.formatMessage({
+            id: `PayoutDetailsForm.personalIdNumberLabel.${country}`,
```

Comment:
We have tried to avoid variables in translation keys. The reason is that most of our customers will have hard time finding correct components if they can't search full translation key from files. (We have used them only in situations where there is dynamic content coming from integrations - like Stripe errors.)

So, chained ternary or `let` should work better here.